### DISCUSSION TITLE
Allow only a certain field to be updated

### DISCUSSION THREAD

**Discussion by shajidhasan** [pocketbase/pocketbase#3882] [Q&A]
Hello! I'm having a blast with Pocketbase. I'm trying to construct a filter to allow only a certain field to be updated, and nothing else. I know how to stop one field from getting updated using the `:isset` modifier, so I can probably use that to disallow updates to all the other fields except one, but that feels convoluted. Is there a simpler way?

I thought of this: `@request.data.*:isset = false && @request.data.myField:isset = true` but this statement doesn't make sense.

**Comment by jaxor24** [comment]
I would like to know this as well

**Comment by suelitonx** [comment]
I believe that unfortunately this functionality is not yet available. But you know, in version 0.34.0 a better way of validating changes in field values ​​was introduced.

```
// (old)
(@request.body.someField:isset = false || @request.body.someField = someField)

// (new)
@request.body.someField:changed = false
```

Obviously, you will need to manually validate all fields.
